BUILD-SAFARI.md

```diff
@@ -1,52 +1,31 @@
-## Build Process
+## Safari App Extension Build Instructions
 
-Prerequisites: Node 12.3.0 or higher, npm 6.9.0 or higher.
+**1. Install Dependencies**
 
-**1. Set the version/build number**
-
-Set the build number in the following files:
-
-* package.json (package version - javascript)
-* dist/manifest.json (package version - Chrome and FF archive)
-* src/safari-app-extension/AlpheiosReadingTools/Info.plist (MacOS App)
-* src/safari-app-extension/AlpheiosSafari/Info.plist (Safari App Extension)
-
-
-**2. Build the distribution Javascript and CSS files**
+**Prerequisites**:
+* Node 13.7.0 or higher
+* Npm 6.13.6 or higher
+* XCode 10.0
```

Comment:
I was using XCode 11.5 to build an extension. I'm not sure if it will work with XCode 10. Maybe we should lift that?

Reply:
ok let me check what I was using too.

Reply:
updated to XCode 11.3.1 and Swfit 5.1.3 since that's what I used for the build.